GVFS/GVFS/CommandLine/MountVerb.cs

```diff
@@ -276,10 +276,6 @@ private bool TryMount(GVFSEnlistment enlistment, string mountExecutableLocation,
             }
 
             const string ParamPrefix = "--";
-            if (GVFSPlatform.Instance.IsUnderConstruction)
-            {
-                mountExecutableLocation = Path.Combine(ProcessHelper.GetCurrentProcessLocation(), "gvfs.mount");

```

Comment:
I think this cleanup makes sense.  Previously it would have came in as "GVFS.Mount" but due to the change in MountExecutableName it'll correctly be "gvfs.mount".

Reply:
Is it lower case now on Windows?

Reply:
Windows is case insensitive. We could certainly change the csproj's to all generate the same casing, but it is not required for this change.